wrapping/python/openmeeg/numpy.i

```diff
@@ -40,7 +40,10 @@
 #define NO_IMPORT_ARRAY
 #endif
 #include "stdio.h"
-#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
```

Comment:
This file is vendored directly from https://github.com/numpy/numpy/blob/main/tools/swig/numpy.i . Can we avoid modifying it?

Reply:
I know, but I do not see how. The only possibility is to forget about requiring the 1.14 numpy API and stick with the 1.7 one. I have no idea who, why and when the 1.14 requirement was added.... The only possibility I can see would be to keep the diff as a patch and apply it, but I'm not sure this is worth for such a small change (we could even remove numpy.i from the repo and always download the latest version before applying the patch).

Reply:
> The only possibility is to forget about requiring the 1.14 numpy API and stick with the 1.7 one

This gets my vote, unless we need 1.14 capabilities for some reason